### DISCUSSION TITLE
clover is half functional

### DISCUSSION THREAD

**Discussion by pterotronic** [CloverHackyColor/CloverBootloader#602] [Q&A]
I couldn't get any response from issues section. Hope someone in discussions can help:

I got the computer to boot to clover. However, no matter what disk you pick in the clover menu, it does not boot. If i boot directly to c:, it boots fine. If I boot to c: from the cover menu. If fails. You just get a blank screen with a cursor blinking in the upper left corner.

I want to boot to nvme ssd in pci-e 3 slot. But if it won't even boot to c:, I think there's something wrong that needs to be fixed first.

the new nvme ssd appears in the clover menu, and in windows explorer. It just doesn't show up in my bios, that's why I'm trying to use clover.

You can select the nvme ssd in the clover menu, but it does the same thing as when you select c: from the clover menu. You just get a blank screen with the cursor blinking in the upper left corner.

thanks for any help w this.

**Comment by MeGaL0DoN** [comment]
Use OpenCore

**Comment by siliconbeaver** [comment]
I had a similar issue. CloverISO-5070. got fixed.

DefaultVolume and GUID.
my problem was that initial GUID was wrong (typo).
I read debug.log carefully. then right copied the GUID. re-start. it boot into OS when timeout.
